diagnostic.md

```diff
@@ -16,13 +16,13 @@ Graces-MacBook-Pro:try grace$
 
 What is the full path to the current directory?
 
-_Replace this text with your answer._
+/Users/grace/wdi/try
 
 ## Question 2
 
 Describe GitHub, **briefly** in your own words.
 
-_Replace this text with your answer._
+Web-based Git repository hosting service
```

Comment:
At a high level yes, more specific 'A user interface (UI) for accessing git's version control capabilities for our project repositories.'